### PR TITLE
v1.4.0_SSR-2337 Investigate storybook for setter, getter

### PR DESCRIPTION
**Pull request by voduchau**

### CODE REVIEW

File: storybook-addons/manager.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  fontWeight: 'bold'
+}
+
+export const GetButton = () => {
+  const [args] = useArgs();
+  function _handleClick() {
+    const iframe = document.getElementById("storybook-preview-iframe");
+    const component = iframe.contentWindow.document.getElementById(args.id);
+    Object.keys(args).forEach(key => {
+      console.log(`"${key}" value: ${component[key]}`);
```
Comment (TomokoMiyake) [inline storybook-addons/manager.js:20]: If applying this script to each component and click "Get Props" button, the console log will be complex and cannot distinguish the component info.
How about making it this kind of object?
DatePicker props' value: { id: "test", value: "12-12", ...}

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]
About storybook-addons directory, do we need to create it? We have .storybook and can we use that??

File: storybook-addons/manager.js (hunk 1 above)
Comment (voduchau) [reply storybook-addons/manager.js]: I updated

File: storybook-addons/manager.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  fontWeight: 'bold'
+}
+
+export const GetButton = () => {
+  const [args] = useArgs();
+  function _handleClick() {
+    const iframe = document.getElementById("storybook-preview-iframe");
+    const component = iframe.contentWindow.document.getElementById(args.id);
+    const componentName = component.nodeName.replace("KUC-","");
+    console.log(`${componentName}'s props value:`, args);
```
Comment (TomokoMiyake) [inline storybook-addons/manager.js:20]: How about changing to this structure if possible?
スクリーンショット 2022-07-14 18 09 21

It looks not good about the starting point of args when opening the object detail.

スクリーンショット 2022-07-14 18 20 17

Comment (TomokoMiyake) [inline storybook-addons/manager.js:20]:
```suggestion
    const componentName = component.nodeName.replace("KUC-","");
    console.log(`${componentName}:`, args);
```

**Review by TomokoMiyake — COMMENTED** [review]
Please check review comment.

**Review by TomokoMiyake — APPROVED** [review]
LGTM!

**Pull request merged**